pinot-common/src/main/java/org/apache/pinot/common/config/FieldConfig.java

```diff
@@ -32,9 +32,12 @@
 
   public static String BLOOM_FILTER_COLUMN_KEY = "field.config.bloom.filter";
   public static String ON_HEAP_DICTIONARY_COLUMN_KEY = "field.config.onheap.dictionary";
-  public static String TEXT_INDEX_REALTIME_READER_REFRESH_KEY = "field.config.realtime.reader.refresh";
   public static String VAR_LENGTH_DICTIONARY_COLUMN_KEY = "field.config.var.length.dictionary";
 
+  // Lucene index properties
+  public static String LUCENE_TEXT_INDEX_REALTIME_READER_REFRESH_KEY = "field.config.text.index.realtime.reader.refresh";
```

Comment:
Not related to this but can we remove the `field.config.` prefix from all these keys?

Reply:
done